ui/src/buckets/components/BucketsTab.tsx

```diff
@@ -78,7 +98,7 @@ class BucketsTab extends PureComponent<Props, State> {
         <FilterList<PrettyBucket>
           searchTerm={searchTerm}
           searchKeys={['name', 'ruleString', 'labels[].name']}
-          list={this.prettyBuckets(buckets)}
+          list={prettyBuckets(buckets)}
```

Comment:
one day in the future, we could memoize this function as well. feel free to merge without it

Reply:
I agree. Okay :)